README.md

```diff
@@ -8,16 +51,238 @@ so don't try to use it for anything close to a production environment.
+also provides a configuration mechanism using a Kubernetes custom resource definition (CRD) that allows the hub cluster administrator to configure the different IMS microservices properly.
+
+
+### <a name='Deploytheoperatoronyourcluster'></a>Deploy the operator on your cluster
+
+The IMS operator is installed on an OpenShift cluster where Red Hat Advanced Cluster Management for Kubernetes (RHACM), a.k.a. hub cluster, is installed too. Let’s install the operator. You can use the latest automatic build from the [openshift-kni namespace](https://quay.io/repository/openshift-kni/oran-o2ims-operator) in quay.io or build your container image with the latest code. 
+
+If you want to build the image yourself and push it to your registry right after:
+
+> :warning: Replace the IMAGE_NAME value with the full name of your container image
+
+```bash
+$ export IMAGE_NAME=quay.io/alosadag/oran-o2ims:v4.16
+$ git clone https://github.com/openshift-kni/oran-o2ims.git
+$ cd oran-o2ims
+$ make docker-build docker-push CONTAINER_TOOL=podman IMG=${IMAGE_NAME}
+
+..REDACTED..
+Update dependencies
+hack/update_deps.sh
+hack/install_test_deps.sh
+Downloading golangci-lint
+…
+[3/3] STEP 5/5: ENTRYPOINT ["/usr/bin/oran-o2ims"]
+[3/3] COMMIT quay.io/alosadag/oran-o2ims:v4.16
+--> eaa55268bfff
+Successfully tagged quay.io/alosadag/oran-o2ims:v4.16
+eaa55268bfffeb23644c545b3d0a768326821e0afea8b146c51835b3f90a9d0c
+```
+
+Now, let's deploy the operator. If you want to deploy your already built image then add the `IMG=${IMAGE_NAME}` argument to the `make` command:
+
+```sh
+$ make deploy install
+
+… REDACTED …
+Update dependencies
+hack/update_deps.sh
+hack/install_test_deps.sh
+Downloading golangci-lint
+… REDACTED …
+/home/alosadag/Documents/CNF/ORAN/oran-o2ims/bin/kustomize build config/default | /home/alosadag/Documents/CNF/ORAN/oran-o2ims/bin/kubectl apply -f -
+namespace/oran-o2ims created
+serviceaccount/oran-o2ims-controller-manager created
+role.rbac.authorization.k8s.io/oran-o2ims-leader-election-role created
+clusterrole.rbac.authorization.k8s.io/oran-o2ims-manager-role created
+clusterrole.rbac.authorization.k8s.io/oran-o2ims-metrics-reader created
+clusterrole.rbac.authorization.k8s.io/oran-o2ims-proxy-role created
+rolebinding.rbac.authorization.k8s.io/oran-o2ims-leader-election-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/oran-o2ims-manager-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/oran-o2ims-proxy-rolebinding created
+configmap/oran-o2ims-env-config created
+service/oran-o2ims-controller-manager-metrics-service created
+deployment.apps/oran-o2ims-controller-manager created
+```
+
+The operator is installed in the `oran-o2ims` namespace, which is created during the install.
+
+```bash
+$ oc get pods -n oran-o2ims-system
+NAME                                             READY   STATUS    RESTARTS   AGE
+oran-o2ims-controller-manager-8668794cdc-27xvf   2/2     Running   0          11s
```

Comment:
Could you take another snapshot of this output?  With the latest code the other servers should be running by default.

Reply:
yes, i guess that's new since I did some tests two days ago and it required to populate the inventory CR to start the different components.